src/templates/nomination_edit.html

```diff
@@ -0,0 +1,42 @@
+{% extends "layout.html" %}
+
+{% block body %}
+
+{% if canedit == true %}
+<h1>{{'Edit a BOF Session'|trans}}</h1>
+
+<form action="{{ path_for( 'updatetopic', {'id': topic.id})}}" method="POST" class="bof-form">
+  <input type="hidden" name="id" value="{{topic.id}}"/>
+  <div class="bof-form__item">
+    <label class="bof-form__label">{{'Title'|trans}}</label>
+    <input type="text" name="title" class="bof-form__input" value="{{topic.name}}"/>
+  </div>
+
+  <div class="bof-form__item">
+    <label class="bof-form__label">{{'Description'|trans}}</label>
+    <textarea name="description" class="bof-form__textarea">{{topic.description}}</textarea>
+  </div>
+
+  <input type="submit" value="{{'Save'|trans}}" class="btn btn--primary" />
+</form>
+
+<a href="/topics"><input type="submit" value="{{'Cancel'|trans}}" class="btn btn--secondary" /></a>
+{% else %}
```

Comment:
A blank line before/after the {%else%} would help it stand out a little more.  When scanning the file, I missed it several times, and at first thought it was just missing. :(

Reply:
good point! I fixed that.